opmd_viewer/openpmd_timeseries/interactive.py

```diff
@@ -116,7 +116,27 @@ def refresh_ptcl(force=False) :
                         species=ptcl_species_button.value, plot=True,
                         vmin=vmin, vmax=vmax, cmap=ptcl_color_button.value,
                         nbins=ptcl_bins_button.value )
-                
+
+        def refresh_species(b):
+            """
+            Refresh the particle species buttons
+            """
+            # Get available records for this species (remove charge and
+            # mass as they are typically less interesting)
+            avail_records = [ q for q in \
+                        self.avail_record_components[ptcl_species_button.value]
+                        if (q in ['charge', 'mass'])==False ]
```

Comment:
let us make this an `exclude_list=...` option of `slider()` in the future so users can choose what to exclude